hardware/Tado.cpp

```diff
@@ -22,26 +22,28 @@
 #include "../json/json.h"
 #include "../webserver/Base64.h"
 #include "Tado.h"
-#include <regex>
-#include <initializer_list>
 
-#define round(a) ( int ) ( a + .5 )
+using namespace std;
+
 const int TADO_POLL_INTERVAL = 30;  // The plugin should collect information from the API every n seconds.
-const std::string TADO_API_ENVIRONMENT_URL = "https://my.tado.com/webapp/env.js";
+const string TADO_API_ENVIRONMENT_URL = "https://my.tado.com/webapp/env.js";
```

Comment:
instead of const int/string in the .cpp, could you use

`#define TADO_POLL_INTERVAL 30`
or

`#define TADO_API_ENVIRONMENT_URL "https://my.tado.com/webapp/env.js"`

Reply:
Changed constants into #define statements.